library/src/main/java/com/google/android/exoplayer2/text/Cue.java

```diff
@@ -134,10 +143,26 @@
   /**
    * The size of the cue box in the writing direction specified as a fraction of the viewport size
    * in that direction, or {@link #DIMEN_UNSET}.
+   * <p>
+   * If {@link #bitmap} is not null then this value is used to indicate the width
```

Comment:
I don't think this comment is needed. Size is the same for text and for the bitmap.